packages/test/src/run-a-worker.ts

```diff
@@ -1,7 +1,16 @@
-import { Worker } from '@temporalio/worker';
+import { Worker, Core, DefaultLogger } from '@temporalio/worker';
 import * as activities from './activities';
 
 async function main() {
+  if (['1', 'y', 'yes', 't', 'true'].includes((process.env.DEBUG ?? '').toLowerCase())) {
```

Comment:
IMO Just checking for existence is much better than this dance. Easy enough to clear an env var if you want to set it to false.

Also just `DEBUG` is too generic I think. Something more temporal specific might be good.

Reply:
FWIW, don't use `TEMPORAL_DEBUG`. That env var is used in Java and Go SDKs to remove the workflow deadlock timer so code can be stepped slowly.